Gemfile

```diff
@@ -85,7 +85,7 @@ group :test, :development do
   # Pretty printed test output
   gem 'turn', '~> 0.8.3', :require => false
   gem 'fuubar'
-  gem 'rspec-rails'
+  gem 'rspec-rails', '~> 2.99.0'
```

Comment:
As I suspected, this was the result of some serious fudging while I was setting up my dev environment. It was due to errors in some tests but in hindsight I don't think the old version of rspec-rails was the cause. To save any headaches I just reverted to the Gemfile from the latest master and checked that everything works ok - it does.